build.gradle

```diff
@@ -142,6 +150,14 @@ if ("true" == project.findProperty('refresh.cache')) {
     }
 }
 
+tasks['testBattery'].doFirst {
+    def batteryPath = new File(createBatteryPath())
+    batteryPath.mkdirs()
+    environment 'BATTERY_TESTS_DETECT_JAR_PATH', createArtifactName()
+    environment 'BATTERY_TESTS_PATH', batteryPath.getCanonicalPath()
+    environment 'BATTERY_TESTS_JAVA_PATH', org.gradle.internal.jvm.Jvm.current().javaExecutable.getCanonicalPath()
+}
+
```

Comment:
If these environment variables aren't set, what kind of logging do we get?

Reply:
The tests Assume.assumeFalse so the tests just won't be run. The Java/Jar paths are not required to run from intellij. If you invoke the gradle task, they will always be set.